jvm/jfr-annotations/pom.xml

```
@@ -22,6 +22,9 @@
     <artifactId>quick-perf-jfr-annotations</artifactId>
 
     <properties>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+        <dependencies.max.jdk.version>17</dependencies.max.jdk.version>
```

@nagyesta 

Would the following change work?

```suggestion
        <maven.compiler.source>1.8</maven.compiler.source>
        <maven.compiler.target>1.8</maven.compiler.target>
        <dependencies.max.jdk.version>1.8</dependencies.max.jdk.version>
```

---

Hi @jeanbisutti ,
it would NOT work AFAIK because the new (9.1.0) version of the flight recorder dependency we are using requires JDK 17. I have seen enforcer complain when I have tried is with lower JDK versions.

As I have seen, there would be a longshot to maybe use 8.0.1 from the same dependency and that is supposed to work with JDK 11 if I am not wrong. Should I try that?

---

Hi @jeanbisutti ,
good news, it was actually working with 1.8 as well when I used the oldest flightrecorder jar from Maven Central.